sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/block/RuleConstant.java

```diff
@@ -53,6 +53,8 @@
 
     public static final int DEFAULT_SAMPLE_COUNT = 2;
     public static final int DEFAULT_WINDOW_INTERVAL_MS = 1000;
+    public static final int DEFAULT_AVG_USED_TOKENS = 0;
```

Comment:
The `DEFAULT_AVG_USED_TOKENS` should not be put into `RuleConstant` as it is supposed to keep all constants for rules. You can put it inside the `DefaultController` directly and keep the original `RuleConstant`.

Reply:
reasonable